tools/DynamoPackagesAnalyzer/Analyzer/IAnalyze.cs

```diff
@@ -0,0 +1,15 @@
+using DynamoPackagesAnalyzer.Models;
+
+namespace DynamoPackagesAnalyzer.Analyzer
+{
+    /// <summary>
+    /// Allows to process a package
+    /// </summary>
+    internal interface IAnalyze
```

Comment:
what is with this interface ?
Are you anticipating many types to implement this interface ?
If it is just one, then I suggest we keep it as simple as possible

Reply:
Initially I thought that we would need to analyze extra files like like dyf, but it seems that is out of the scope, I'll do that change